### ISSUE TITLE
Unable to release Google Cloud Print jobs

### ISSUE THREAD

**Issue by jfrsw** [Libki/libki-server#132]
With the version currently on master branch, it is no longer possible to release a print job to the Google Cloud Print backend.

When a print job is sent, it appears correctly on the 'Prints' tab of the admin interface. When the 'Print' button is pressed to release the print job, the result appears on a green Success notification, but the message complains about an error analysing the print job ticket. After that, the log keeps outputting lines like that:
"[libki] [FATAL] Use of uninitialized value in lc at /app/lib/Libki/Controller/Administration/API/PrintJob.pm line 240."

Those problems seem to have started with the last updates related to number of print job copies. Will try to debug the code and see if I can send a PR to try and fix it.

**Comment by kylemhall** [comment]
That definitely sounds like the response from the GCP API is not what is expected. The response should be JSON. I'd recommend you use Data::Dumper to print `$response->decoded_content` to the logs to see what you are getting back instead.

**Comment by jfrsw** [comment]
Following your advice, I have dumped the contents of $response->decoded_content.
It turns out it is actually a JSON response, but the message complains about the print ticket:
```
   'message' => "Error al analizar el ticket del trabajo de impresi\x{f3}n",
   'success' => bless( do{\(my $o = 0)}, 'JSON::PP::Boolean' ),
   'request' => {
                  'user' => 'xxxxxx@xxxxx.xxx',
                  'users' => [
                               'xxxxxx@xxxxx.xxx'
                             ],
                  'time' => '0',
                  'params' => {
                                'printerid' => [
                                                 'xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx'
                                               ],
                                'ticket' => [
                                              '{"version":"1.0","print":{"copies":1}}'
                                            ],
                                'title' => [
                                             'print-test-2.pdf'
                                           ]
                              }
                },
   'errorCode' => 424
 };
```
Investigating that error code, I came to <https://developers.google.com/cloud-print/docs/cdd#cjt-example>,
where it shows an example of a sample ticket. On that ticket, the number of copies is specified as
```
{
  "version": "1.0",
  "print": {
    "copies":{"copies" :1}
   }
}
```

Instead of

```
{
  "version": "1.0",
  "print": {
    "copies":1
    }
}
```

which the current server code is sending.

I will try to modify the ticket generation part on the server and if the solution works will create a PR with the patch.

**Comment by loidor** [comment]
I'll close this since it's fixed by #135.
